lib/rubocop/cop/documentation.rb

```diff
@@ -8,7 +8,7 @@ module Documentation
 
       # @api private
       def department_to_basename(department)
-        "cops_#{department.downcase}"
+        "cops_#{department.to_s.downcase.tr('/', '_')}"
```

Comment:
Is `to_s` needed?

Reply:
Yes. `department` is surprisingly a Symbol. `downcase` (not less surprisingly to me) works on Symbol, but `tr` doesn't.
I don't have a strong preference where to add `.to_s`, before or after `downcase`.
Also, I'll be happy to extract this transformation to a variable for better readability if you see fit.

Reply:
I get it! This is fine, thank you for your explanation.